src/test/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/XGBoostEvidenceFilterUnitTest.java

```diff
@@ -0,0 +1,427 @@
+                ++lineNumber;
+                if(line.startsWith("#")) {
+                    continue;
+                }
+                final String[] splitLine = line.split("\t");
+                final String contig = splitLine[0];
+                final Integer end = Integer.parseInt(splitLine[2]) + 1;
+                if(!contigLengthMap.containsKey(contig) || contigLengthMap.get(contig) < end) {
+                    contigLengthMap.put(contig, end);
+                }
+            }
+        } catch(IOException e) {
+            throw new GATKException("Unable to read genome gaps file \"" + svGenomeGapsFile + "\" line " + lineNumber
+                                    + ": " + e.getMessage());
+        }
+
+        SAMFileHeader header = new SAMFileHeader();
+        header.setSortOrder(SAMFileHeader.SortOrder.coordinate);
+        SAMSequenceDictionary dict = new SAMSequenceDictionary();
+        // make up some sequence records
+        for (SortedMap.Entry<String,Integer> contigEntry : contigLengthMap.entrySet()) {
+            final String contig = contigEntry.getKey();
+            final int chromosomeSize = contigEntry.getValue();
+            SAMSequenceRecord rec = new SAMSequenceRecord(contig, chromosomeSize);
+            rec.setSequenceLength(chromosomeSize);
+            dict.addSequence(rec);
+        }
+        header.setSequenceDictionary(dict);
+        return header;
+    }
+
+    private static ReadMetadata initMetadata() {
+        final ReadMetadata.PartitionBounds[] partitionBounds = new ReadMetadata.PartitionBounds[3];
+        partitionBounds[0] = new ReadMetadata.PartitionBounds(0, 1, 0, 10000, 9999);
+        partitionBounds[1] = new ReadMetadata.PartitionBounds(0, 10001, 0, 20000, 9999);
+        partitionBounds[2] = new ReadMetadata.PartitionBounds(0, 20001, 0, 30000, 9999);
+        return new ReadMetadata(Collections.emptySet(), artificialSamHeader,
+                new LibraryStatistics(cumulativeCountsToCDF(featuresTestData.template_size_cumulative_counts),
+                        60000000000L, 600000000L, 1200000000000L, 3000000000L),
+                partitionBounds, 100, 10, featuresTestData.coverage);
+    }
+
+    private static IntHistogram.CDF cumulativeCountsToCDF(final long[] cumulativeCounts) {
+        final float[] cdfFractions = new float[cumulativeCounts.length];
+        final long totalObservations = cumulativeCounts[cumulativeCounts.length - 1];
+        for(int index = 0; index < cdfFractions.length; ++index) {
+            cdfFractions[index] = cumulativeCounts[index] / (float)totalObservations;
+        }
+        return new IntHistogram.CDF(cdfFractions, totalObservations);
+    }
+
+    @Test(groups = "sv")
+    protected void testLocalXGBoostClassifierAccuracy() {
+        // check accuracy: predictions are same as classifierAccuracyData up to tolerance
+        AssertJUnit.assertArrayEquals("Probabilities predicted by classifier do not match saved correct answers",
```

Comment:
Use `testNG` instead of `JUnit` here and elsewhere.

Reply:
This was a subpackage of testNG. I've switched from testng.AssertJUnit to testing.Assert and wrote my own assertArrayEquals()